protocols/kad/src/kbucket.rs

```diff
@@ -293,11 +293,18 @@ where
     }
 
     /// Returns an iterator to all the peer IDs in the bucket, without the pending nodes.
-    pub fn entries_not_pending(&self) -> impl Iterator<Item = (&TPeerId, &TVal)> {
+    pub fn entries_not_pending(&self) -> impl Iterator<Item = &TPeerId> {
```

Comment:
Why change this? It is a strictly less good version.

Reply:
Well, I changed this in one of the intermediate steps, and it turns out that the only function consuming it was actually discarding the value, so it didn't look harmful. But yes, as it is right now, it is a poorer version.